src/components/projectSelect/index.tsx

```diff
@@ -108,16 +116,12 @@ export const ProjectSelect: React.VFC<ProjectSelectProps> = ({
       }}
       optionRenderer={(projectGroup, onClick) => (
         <ProjectOptionGroup
-          key={projectGroup.name}
+          key={projectGroup.groupDisplayName}
           projects={projectGroup.projects}
```

Comment:
Good catch! I noticed this is because staging contains a lot of unnamed projects that use the empty string as their identifier. This isn't something we'll see in prod so I think it's okay to keep this key!